Sources/GRPC/Version.swift

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+internal struct Version {
+  /// The major version.
+  internal static let major = 1
+
+  /// The minor version.
+  internal static let minor = 0
+
+  /// The patch version.
+  internal static let patch = 0
```

Comment:
How are we going to enforce updating this?

Reply:
I was also wondering about this. I'm not really sure at the moment because we have to update this before tagging a release, and anything that automates this check needs to know when a release is happening first.

One idea was to validate in CI against git tags, of course that only works after-the-fact so is of limited use, you can at least update it in a patch release straight after tough if the tag was wrong.

A release checklist has the problem as we have now the same as now: you need to remember to look at it. 

The other idea I had was to build a check into the release notes drafter since I always run that before a release, definitely a hack though.

Any other ideas?

Reply:
I think that hack may be what we need to do for now, sadly.